app/webpacker/css/admin_v3/components/navigation.scss

```diff
@@ -75,7 +75,10 @@ nav.menu {
       display: block;
       padding: 25px 5px;
       color: $color-1 !important;
-      text-transform: uppercase;
+      text-transform: lowercase;
+      &::first-letter {
+        text-transform: uppercase;
+      }
```

Comment:
Neat trick! From what I can see, this is necessary to convert "Order Cycles" to "Order cycles" and the same for "Bulk Order Management".
Why not just update the title source itself?

Hmm that took a few mins to find, I didn't realise it was quite so complicated. But still this looks like a better place to update: https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/helpers/spree/admin/navigation_helper.rb#L27-L29

I think we can use a different string helper other than `.titleize`